### DISCUSSION TITLE
Preload the README to improve load time of crate pages

### DISCUSSION THREAD

**Discussion by lovasoa** [rust-lang/crates.io#4328] [Feature Requests]
Here is a lighthouse report for a package page on crates.io : 

![image](https://user-images.githubusercontent.com/552629/48157886-329d1e00-e2d1-11e8-8dcc-a2decf89593d.png)

A time-to-interactive of 8.6 seconds on mobile makes the site difficult to use.

**Comment by lovasoa** [comment]
I think something to prioritize is to [preload](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content) the readme. It should be easy to do and bring a big benefit in terms of page load time.

**Comment by carols10cents** [comment]
Most of this is likely a duplicate of #204, using ember fastboot. Preloading the readme is an interesting, independent idea, so I'm going to leave this issue open for exploring that possibility.
